hw/hw03/hw 03 sanya1213

```diff
@@ -0,0 +1 @@
+
```

Comment:
zen = str("Beautiful is better than ugly.\nExplicit is better than implicit.\n\
Simple is better than complex.\nComplex is better than complicated.\n\
Flat is better than nested.\nSparse is better than dense.\n\
Readability counts.\nSpecial cases aren't special enough to break the rules.\n\
Although practicality beats purity.\nErrors should never pass silently.\n\
Unless explicitly silenced.\nIn the face of ambiguity, refuse the temptation to guess.\n\
There should be one-- and preferably only one --obvious way to do it.\n\
Although that way may not be obvious at first unless you're Dutch.\n\
Now is better than never.\nAlthough never is often better than *right* now.\n\
If the implementation is hard to explain, it's a bad idea.\n\
If the implementation is easy to explain, it may be a good idea.\n\
Namespaces are one honking great idea -- let's do more of those!")